.github/workflows/build-test-ubuntu-arm64.yml

```diff
@@ -31,6 +31,7 @@ jobs:
     runs-on: [self-hosted, linux-arm64]
     container:
       image: ${{ matrix.docker-image }}
+      options: ${{ matrix.container-options }}
```

Comment:
why?

Reply:
need to install packages for build stubs (similar to pip-build workflow)